languages.yaml

```diff
@@ -2851,6 +2851,39 @@ TypeScript:
       - "#FFFFFF"
     chip: "#2B7489"
   icon: '\u{E69D}'
+Typst:
+  type: markup
+  ascii: |
+    {0}ttttttttttttttttttttttttttttttttttttttt
+    {0}ttttttttttttttttttttttttttttttttttttttt
+    {0}ttttttttttttttttttttttttttttttttttttttt
+    {0}ttttttttttttttttttttttttttttttttttttttt
+    {0}ttttttttttttttt{1}tttt{0}tttttttttttttttttttt
+    {0}ttttttttttttt{1}tttttt{0}tttttttttttttttttttt
+    {0}tttttttttt{1}ttttttttttttttt{0}tttttttttttttt
+    {0}tttttttttt{1}tttttttttttttt{0}ttttttttttttttt
+    {0}ttttttttttttt{1}tttttt{0}tttttttttttttttttttt
+    {0}ttttttttttttt{1}tttttt{0}tttttttttttttttttttt
+    {0}ttttttttttttt{1}tttttt{0}tttttttttttttttttttt
+    {0}ttttttttttttt{1}tttttt{0}tttttttttttttttttttt
+    {0}ttttttttttttt{1}tttttt{0}tttttttttttttttttttt
+    {0}ttttttttttttt{1}tttttt{0}tttttttttttttttttttt
+    {0}ttttttttttttt{1}tttttt{0}tttttttttttttttttttt
+    {0}ttttttttttttt{1}ttttttt{0}tttttttt{1}t{0}tttttttttt
+    {0}tttttttttttttt{1}tttttttttttttt{0}ttttttttttt
+    {0}tttttttttttttttt{1}tttttttttt{0}ttttttttttttt
+    {0}ttttttttttttttttttttttttttttttttttttttt
+    {0}ttttttttttttttttttttttttttttttttttttttt
+    {0}ttttttttttttttttttttttttttttttttttttttt
+    {0}ttttttttttttttttttttttttttttttttttttttt
+  colors:
+    ansi:
+      - cyan
+      - white
+    hex:
+      - "#239DAD"
```

Comment:
My bad, you need to add the hex value for white as the second color

Reply:
No worries! Should be good now.